collections/_posts/2024-04-09-designs-on-artifacthub.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 📢 Meshery Designs Now Supported on Artifact Hub!
+subheading: Working with Meshery Designs is easier than ever  
+date: 2024-04-24
+author_contributor: Chris Carrier
+categories: 
+  - Meshery
+  - OpenSource
+permalink: /blog/designs-on-artifact-hub
+featured-image: /assets/images/posts/2024-04-09-designs-on-artifacthub/meshery-designs-on-artifact-hub-featured.png
+---
+
+As a proud contributor to the Meshery project, I’m thrilled to announce that Artifact Hub has officially recognized Meshery Designs as a native artifact kind. 🎉 Meshery Designs allow you to create, manage, and deploy complex architectures seamlessly, which make them a great addition to the growing collection of Artifact Hub artifacts such as Helm charts, Argo templates, and <a href="https://artifacthub.io/docs/topics/repositories/" target="_blank">many more</a>.
+
+<a href="https://artifacthub.io" target="_blank"><img alt="Artifact Hub Kinds" src="/assets/images/posts/2024-04-09-designs-on-artifacthub/meshery-designs-on-artifact-hub.png" width="50%" align="right" /></a>
+
+## Artifact Hub's first Meshery Designs
+
+[Meshery Catalog](/catalog) is the first repository to have it's Meshery Designs listed on Artifact Hub. Meshery Catalog functions much like a cloud marketplace, providing a user-friendly interface for browsing, discovering, and sharing configurations and patterns for cloud native infrastructure.
+
+Learn more about [Meshery Catalog](https://docs.meshery.io/concepts/catalog).
+
+You can now browse Meshery Designs in the [Artifact Hub search](https://artifacthub.io/packages/search?kind=24&sort=relevance&page=1).
```

Comment:
@leecalcote I see some designs in Artifact Hub so I added the link to the post here.

Reply:
@ctcarrier, most excellent! :)

If we haven't done so in this post, perhaps, let's plan for a follow up post in which we point out artifacthub-pkg.yaml support pertinent to any design export and/or answer "How to make your  Meshery Designs available in Artifact Hub".